code/style.css

```diff
@@ -1,4 +1,228 @@
+*, *:before, *:after {
+  box-sizing: inherit;
+  margin: 0px;
+  padding: 0px;
+}
+
 html {
-  background: #0025ff;
-  color: #fff;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: lightpink;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+header {
+  width: cover;
+  background-color: red;
+  height: 100px;
+}
+
+.navigation-container {
+  background-position: center;
+  height: 100px;
+  width: 92%;
+  margin: auto;
+}
+
+.navigation { 
+  height: 85px;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  margin-top: 15px;
+}
+
+.navigation-logo-img {
+  height: 30px;
+  margin-left: 3%;
+}
+
+.navigation-list {
+  display: none;
+}
+
+.navigation-list a {
+  text-decoration: none;
+  color: white;
+}
+
+.navigation-list-item {
+  font-family: 'Inter', sans-serif;
+  font-size: 18px;
+  display: flex;
+}
+
+.big-news-parent {
+  width: 96%;
+  margin: auto;
+  background-color: white;
+  padding: 20px 0px 20px 0px;
+  display: flex;
+  align-content: flex-start;
+  justify-content: center;
+  flex-wrap: wrap;
+}
+
```

Comment:
How did you ad the small blue line att the top att the news cards? NIce done!

Reply:
With the border-top property! News cards with `class="red"` for example gets this `border-top: 4px red solid;` (copied from line 148 in the CSS).